### PR TITLE
4091 My resources performance improvements

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Checkout develop branch and run hydroshare.
2. Create 200 composite resources (use a management command or run commands in Django shell for this)
3. Load the my resources page and keep a note of the page load time.
4. Checkout branch 4091-my-resources-performance.
5. Reload the my resources page. Verify that the page load time is less (by about 30%) compared to what you noted in step 3.
6. You can also verify the page load time improvement (slight improvement) for the profile page.

### CODE REVIEW

File: theme/templates/accounts/profile.html [hunk 1]
```diff
@@ -730,30 +731,32 @@ <h3 class="panel-title"><span
...
+                                                    {% if abstract %}
+                                                        <div class="activity-description col-sm-12">
+                                                            <p>
+                                                                <strong>ABSTRACT: </strong>{{ abstract|linebreaks }}
+                                                            </p>
+                                                        </div>
+                                                        <div class="btn-show-more toggle-abstract btn btn-default btn-xs">Show More</div>
+                                                    {% endif %}
+                                               {% endwith %}
+                                           {%  endwith %}
```
Comment (devincowan) [inline theme/templates/accounts/profile.html:759]:
```suggestion
                                           {% endwith %}
```

**Review by devincowan — COMMENTED** [review]
looks good!
A couple small formatting changes and one question for you...

File: hs_core/views/utils.py [hunk 2]
```diff
@@ -624,21 +627,18 @@ def get_my_resources_list(user, annotate=True):
             When(short_id__in=labeled_resources.values_list('short_id', flat=True),
                  then=Value(True, BooleanField()))))
 
-        resource_collection = resource_collection.only('short_id', 'title', 'resource_type', 'created')
+        resource_collection = resource_collection.only('short_id', 'title', 'resource_type', 'created', 'content_type')
```
[the remark this answers is not in the document]
Comment (pkdash) [reply hs_core/views/utils.py]: Not needed. Removed.

File: theme/templates/accounts/profile.html [hunk 3]
```diff
@@ -315,29 +315,30 @@ <h3 class="panel-title"><span class="glyphicon glyphicon-time">&nbsp;</span>Rece
...
+                                                                </p>
+                                                            </div>
+                                                            <div class="btn-show-more toggle-abstract btn btn-default btn-xs">Show More</div>
+                                                        {% endif %}
+                                                    {% endwith %}
                                                     </div>
-                                                    <div class="btn-show-more toggle-abstract btn btn-default btn-xs">Show More</div>
-                                                {% endif %}
-                                            </div>
+                                                    {% endwith %}
```
[the remark this answers is not in the document]
Comment (pkdash) [reply theme/templates/accounts/profile.html]: fixed

File: theme/templates/accounts/profile.html (hunk 1 above)
Comment (pkdash) [reply theme/templates/accounts/profile.html]: fixed

**Review by devincowan — APPROVED** [review]

**Pull request merged**